packages/site/src/layout/NavMenu.module.css

```diff
@@ -28,7 +28,8 @@
   display: flex;
   flex-direction: column;
   gap: var(--space-large);
-  overflow-y: scroll;
+  overflow-y: auto;
+  scrollbar-gutter: stable;
```

Comment:
`overflow-y: auto;` alone makes the NavMenu really jumpy when opening and closing long disclosures. The Search input was jumping in size also.
`scrollbar-gutter: stable;` ensures space is always reserved for the scrollbar, even when it’s not visible.